docs/tutorial/GetStarted.md

```diff
@@ -323,11 +327,109 @@ The service `name`, `cpus` and `mem` are populated from the default values in th
 
 If you need further examples, you can refer to the [repo/packages/H/hello-world](/repo/packages/H/hello-world) package.
 
+### Step 3.1 : DC/OS Integration
+
+By default, a DC/OS service is deployed on a [private agent node](https://dcos.io/docs/1.9/overview/concepts/#private-agent-node). To allow configuration control or monitoring of a service by a user, the Admin Router can act as a reverse proxy by proxying calls on the master node to the service in a private node on the cluster.
+
+The Admin Router currently supports only one reverse proxy destination. This step is optional, if you don't want to expose your service endpoint, you can skip to the next step.
+
+#### Service Endpoints
+
+The Admin Router allows Marathon tasks to define custom service UI and HTTP endpoints, which are made available as `/service/<service-name>`. Set the following Marathon task labels to enable this:
+
+```json
+"labels": {
+    "DCOS_SERVICE_NAME": "<service-name>",
+    "DCOS_SERVICE_PORT_INDEX": "0",
+    "DCOS_SERVICE_SCHEME": "http"
+  }
+```
+
+To enable the forwarding to work reliably across task failures, we recommend co-locating the endpoints with the task. This way, if the task is restarted on another host and with different ports, Admin Router will pick up the new labels and update the routing. Note: Due to caching, there can be an up to 30-second delay before the new routing is working.
+
+We recommend having only a single task setting these labels for a given service name. If multiple task instances have the same service name label, Admin Router will pick one of the task instances deterministically, but this might make debugging issues more difficult.
+
+Since the paths to resources for clients connecting to Admin Router will differ from those paths the service actually has, ensure the service is configured to run behind a proxy. This often means relative paths are preferred to absolute paths. In particular, resources expected to be used by a UI should be verified to work through a proxy.
+
+Tasks running in nested [Marathon app groups](https://mesosphere.github.io/marathon/docs/application-groups.html) will be available only using their service name (i.e., `/service/<service-name>`), not by the Marathon app group name (i.e., `/service/app-group/<service-name>`).
+
+
+#### Health Checks
+
+Service health check information can be surfaced in the DC/OS services UI tab by defining one or more healthChecks in the Service’s Marathon template. For example:
+
+```
+"healthChecks": [
+   {
+       "path": "/",
+       "portIndex": 0,
+       "protocol": "HTTP",
+       "gracePeriodSeconds": 5,
+       "intervalSeconds": 60,
+       "timeoutSeconds": 10,
+       "maxConsecutiveFailures": 3
+   }
+]
+```
+
+You can learn more about `healthChecks` parameter at [Marathon API](https://mesosphere.github.io/marathon/docs/health-checks.html) page.
```

Comment:
See the [health checks documentation](https://dcos.io/docs/1.9/deploying-services/creating-services/health-checks/) for more information.